src/webgpu/util/math.ts

```diff
@@ -565,6 +566,23 @@ export function fullF32Range(
   return bit_fields.map(hexToF32);
 }
 
+/**
+ * @returns an ascending sorted array of numbers.
+ *
+ * The numbers returned are based on the `full32Range` as described above. The difference comes depending
+ * on the `source` parameter. If the `source` is `const` then the numbers will be restricted to be
+ * in the range `[low, high]`. This allows filtering out a set of `f32` values which are invalid for
+ * const-evaluation but are needed to test the non-const implementation.
+ *
+ * @param source the input source for the test. If the `source` is `const` then the return will be filtered
+ * @param low the lowest f32 value to permit when filtered
+ * @param high the highest f32 value to permit when filtered
+ */
+export function sourceFilteredF32Range(source: String, low: number, high: number): Array<number> {
+  const interval = new F32Interval(low, high);
```

Comment:
self-nit: I know I suggested this, but now seeing it in situ, I am leery of pulling in F32Interval here, since a bunch of stuff in f32_interval.js depends on math.js and I have previously hit issues with weird dependency issues.

Might be better to just replace interval.contains(x) with (x >= low && x <= high), which also avoids building a F32Interval and a function call

Reply:
done